### ISSUE TITLE
prune the number of image interpolations and replace with new material 

### ISSUE THREAD

**Issue by jimustafa** [matplotlib/cheatsheets#81]
Based on https://github.com/matplotlib/cheatsheets/pull/80#issuecomment-955173036, there may be an opportunity to reduce the number of image interpolation methods that are demonstrated, and make room for new material.

**Comment by rougier** [comment]
I agree that a 3x3 (9 filters) would be enough and we'll have some space for other stuff. Which filters do we select ? I would go for: None, nearest, bilinear, bicubic, spline16, catrom, mitchell, gaussian, lanczos

**Comment by jklymak** [comment]
Well, matplotlib uses 'hanning' by default when downsampling now, so maybe should include that ;-)
